color-picker-project/src/components/colorpicker/ColorPicker.css

```diff
@@ -19,10 +19,11 @@
 
 .color-picker-color {
     margin-bottom: 3rem;
+    width: '70%'
```

Comment:
I don't think percentage values work in css if they're wrapped in quotes? I think whatever styles are actually being used for this class are currently just coming from the media queries.

Reply:
changed.